spec/models/organization_spec.rb

```
@@ -65,7 +65,7 @@ def get_organization
     end
 
     it 'Destroys users and owner as well' do
-      organization = Organization.new(quota_in_bytes: 1234567890, name: 'wadus', seats: 5).save
+      organization = Organization.new(quota_in_bytes: 123456789000, name: 'wadus', seats: 5).save
```

Same as above, Sequel 4 has better error handling and we were actually going over quota when creating users but Sequel was swallowing the errors.